File: java/Fibonacci.java
public class Fibonacci {
    public static void main(String[] args) {
        printFiboSeries(10);
    }

    public static void printFiboSeries(int limit){
        int first =0;
        int second =1;

        for (int i =0; i<=limit;i++){
            System.out.println(first);
            int next = first + second;
            first= second;
            second=next;
        }
    }
}

File: js/fibonacci.js
function getFibonacci(limit){
    var first =0;
    var second =1;

    for(let i =0; i<=limit; i++){
        console.log(first);
        var next = first + second;
        first = second;
        second = next;
    }
}

getFibonacci(10);